File: Dashboard/examples/License.cs
using System;
using System.Net;
using System.IO;

namespace CurlExample
{
    class Program
    {
        static void Main(string[] args)
        {
            string url = "http://<your_ip>:<port>/api/client";
            string apiKey = "API_KEY";
            string license = "<LICENSE>";
            string product = "<PRODUCT>";
            string version = "<VERSION>";

            string postData = "license=" + license + "&product=" + product + "&version=" + version;

            HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
            request.Method = "POST";
            request.Headers.Add("Authorization", apiKey);

            byte[] data = System.Text.Encoding.UTF8.GetBytes(postData);
            request.ContentType = "application/x-www-form-urlencoded";
            request.ContentLength = data.Length;

            Stream stream = request.GetRequestStream();
            stream.Write(data, 0, data.Length);
            stream.Close();

            HttpWebResponse response = (HttpWebResponse)request.GetResponse();
            string responseString = new StreamReader(response.GetResponseStream()).ReadToEnd();
            Console.WriteLine(responseString);
        }
    }
}

File: Dashboard/examples/License.kt
import java.net.HttpURLConnection
import java.net.URL
import java.net.URLEncoder

fun main(args: Array<String>) {
    val url = URL("http://<your_ip>:<port>/api/client")
    val apiKey = "API_KEY"
    val license = "<LICENSE>"
    val product = "<PRODUCT>"
    val version = "<VERSION>"

    val postData = "license=${URLEncoder.encode(license, "UTF-8")}&product=${URLEncoder.encode(product, "UTF-8")}&version=${URLEncoder.encode(version, "UTF-8")}"

    val connection = url.openConnection() as HttpURLConnection
    connection.requestMethod = "POST"
    connection.setRequestProperty("Authorization", apiKey)
    connection.setRequestProperty("Content-Type", "application/x-www-form-urlencoded")
    connection.doOutput = true

    val outputBytes = postData.toByteArray(charset("UTF-8"))
    connection.outputStream.write(outputBytes)

    val responseCode = connection.responseCode
    if (responseCode == HttpURLConnection.HTTP_OK) {
        val responseBytes = connection.inputStream.readBytes()
        val response = String(responseBytes)
        println(response)
    } else {
        println("Failed with HTTP code: $responseCode")
    }
    connection.disconnect()
}
